linux_os/guide/services/avahi/disable_avahi_group/package_avahi_removed/rule.yml

```diff
@@ -0,0 +1,42 @@
+documentation_complete: true
+
+prodtype: sle12,sle15
+
+title: 'Uninstall avahi Server Package'
+
+description: |-
+    If the system does not need to have an Avahi server which implements 
+    the DNS Service Discovery and Multicast DNS protocols,
+    the avahi-autoipd and avahi packages can be uninstalled.
+
+rationale: |-
+    Automatic discovery of network services is not normally required for 
+    system functionality. It is recommended to remove this package to reduce 
+    the potential attack surface.
+
+severity: medium
+
+identifiers:
+    cce@sle12: CCE-92314-4
+    cce@sle15: CCE-92464-7
+    
+references:
```

Comment:
Are you sure that all these references are valid for these new rules? Have you confirmed them? If not, I would suggest to keep only the confirmed references.

Reply:
Regarding the references for these new rules, are you sure that all these references are related to these new rules? It seems these references were copied from similar rules. I just would like to know if you confirmed all of them. Otherwise, I recommend to keep only the references you are sure about.